api/src/pcapi/routes/backoffice_v3/utils.py

```diff
@@ -196,7 +196,7 @@ def generate_search_query(
     search_parameters: typing.Iterable[dict[str, typing.Any]],
     fields_definition: dict[str, dict[str, typing.Any]],
     joins_definition: dict[str, list[tuple[str, tuple]]],
-    operators_definition: dict | None = None,
+    operators_definition: dict[str, dict[str, typing.Any]] | None = None,
```

Comment:
Suggestion (facultatif): je ne suis pas certain que ce soit mieux en pratique mais on pourrait essayer ce genre de typage

```suggestion
    operators_definition: dict[str, dict[typing.Callable, bool]] | None = None,
```